nextflow_schema.json

```
@@ -97,6 +138,23 @@
                     "description": "Specify the distance at which the annotation script decides if a candidate is a circRNA or EI-circRNA.",
                     "help_text": "During annotation, if one of the start or end position of a circular candidate imperfectly overlaps an exon boundary, the script will consider positions within 'exon_boundary' (default 0bp) to be an exonic circRNA. If they fall outside of this range, the candidate is assumed to be an exonic-intronic circRNA, and the entire underlying sequence is taken for miRNA analysis, as opposed to just the exonic sequences for canonical exonic circRNAs. ",
                     "default": 0
+                },
+                "mirna_tools": {
+                    "type": "string",
+                    "fa_icon": "fas fa-wrench",
+                    "description": "Comma separated list of miRNA bindingsite prediction tools to use. Supported tools: miranda, targetscan.",
+                    "help_text": "Select one or a combination of miRNA bindingsite prediction tools for the pipeline e.g:\n--tool 'miranda, targetscan'",
```

I now it works technically, but I would prefer not to guide users towards using spaces in parameters. This works when configuring the pipeline via `.config` files, but it will not work when used directly in the command line, because the space will be interpreted as a delimiter.
```suggestion
                    "help_text": "Select one or a combination of miRNA bindingsite prediction tools for the pipeline e.g:\n--mirna_tools 'miranda,targetscan'",
```

---

Also change this in `nextflow.config` and in the `default` field

---

Ok it might work when using quotes but still xD